### ISSUE TITLE
can javacpp-presets ffmpeg support ios?

### ISSUE THREAD

**Issue by dongguadan** [bytedeco/javacpp-presets#235]
Excuse me. i have a question that how to use ffmpeg of javacpp-presets on ios?  well i have try it successfully on win10 and mac.

**Comment by saudet** [comment]
Sure, it's possible. Would you like to work on that?

**Comment by dongguadan** [comment]
Yeah. i would like to work on that  and  i want to reuse it on centos 、windows、android and ios. But i saw in the script only support mac but not ios. Can you give me a suggestion?  thank you very much

**Comment by saudet** [comment]
Just add an entry for ios-arm...

**Comment by saudet** [comment]
RoboVM is dead now (https://github.com/bytedeco/javacpp-presets/issues/259) and the next best option is Intel Multi-OS Engine: https://software.intel.com/en-us/multi-os-engine It's basically Android running on iOS. If you could start checking this out and see what works and what doesn't it would help a lot!

**Comment by saudet** [comment]
And RoboVM is alive again, but now we also have [Gluon VM](http://gluonhq.com/products/mobile/vm/ ) that supports Java 9 and Android too! 
/cc @johanvos  

@dongguadan, it was pretty straightforward to get OpenBLAS builds for iOS: https://github.com/bytedeco/javacpp-presets/commit/53e93263a9cafe36af0a7afd872233d3318b7819
Could you give it a try by yourself for FFmpeg? If you encounter any issues, I will help you out! Thanks
